pkg/openthread/Makefile.include

```diff
@@ -8,3 +8,9 @@ ifneq (,$(filter openthread_contrib,$(USEMODULE)))
   DIRS += $(OPENTHREAD_DIR)/contrib
   DIRS += $(OPENTHREAD_DIR)/contrib/netdev
 endif
+
+ifneq (,$(filter cortex-m0%,$(CPU_ARCH)))
```

Comment:
This one should also be disabled for `cortex-m3` it is failing when running `buildtest` on the iotlab/fox boards.